components/anchor/index.en-US.md

```diff
@@ -20,6 +20,8 @@ For displaying anchor hyperlinks on page and jumping between them.
 | offsetTop    | Pixels to offset from top when calculating position of scroll | number | 0 |
 | offsetBottom | Pixels to offset from bottom when calculating position of scroll | number | - |
 | bounds     | Bounding distance of anchor area | number | 5(px) |
+| affix | Fixed mode of Anchor | boolean | false |
+| inkInFixed | Whether show ink-balls in Fixed mode | boolean | false |
```

Comment:
`showInk` seems telling users that they could hide ink in all modes when set it false..